### PR TITLE
[Bug] Fix automation script path issue

### PR DESCRIPTION
**Pull request by Zzm0809**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: script/bin/init_tools_main.sh [hunk 1]
```diff
@@ -295,9 +347,44 @@ while true; do
...
+             # Use source transparent transmission in debug mode
+            source "${APP_HOME}"/bin/auto.sh restart
+            break
+            ;;
+        no | n )
+            echo_warning_msg
+            echo
+            echo -e "${GREEN}The Dinky service startup script has been skipped, 请先执行上述命令，然后手动启动服务 -> ${APP_HOME}/bin/auto.sh restart | start。${RESET}"
+            break
+            ;;
```
Comment (aiwenmo) [inline script/bin/init_tools_main.sh]: Use English.

File: script/bin/parse_yml.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# 检查参数个数是否正确
+if [ $# -ne 2 ]; then
+    echo "Please enter the parameters in the correct format, the format is: $0 yaml_file_path key"
+    exit 1
+fi
```
Comment (aiwenmo) [inline script/bin/parse_yml.sh]: Use English.

File: script/bin/parse_yml.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+# Used to store the finally found value
+value=""
+
+# Process the file first and remove comment lines (assuming the comment starts with #) and blank lines
+temp_file=$(mktemp)
+grep -Ev '^(#|$)' "$yaml_file_path" > "$temp_file"
+
+# 开始按层级查找值
+current_data=$(cat "$temp_file")
+for part in "${key_parts[@]}"; do
```
Comment (aiwenmo) [inline script/bin/parse_yml.sh]: Use English.

**Review by aiwenmo — COMMENTED** [review]
Use English.

**Comment by Zzm0809** [comment]
@aiwenmo PTAL

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**